src/encoded/static/components/experiment_series.js

```diff
@@ -171,12 +169,13 @@ const ExperimentSeriesComponent = (props, reactContext) => {
                                     </div>
                                 : null}
 
-                                {terms.length > 0 || speciesRender ?
+                                {(context.biosample_summary && context.biosample_summary.length > 0) || speciesRender ?
                                     <div data-test="biosamplesummary">
                                         <dt>Biosample summary</dt>
                                         <dd>
-                                            {terms.length > 0 ? <span>{terms.join(' and ')} </span> : null}
-                                            {speciesRender ? <span>({speciesRender})</span> : null}
+                                            {speciesRender ? <span>{speciesRender}</span> : null}
+                                            &nbsp;
```

Comment:
This inserts a non-breaking space even if `speciesRender` is null/undefined. That would be visible as an odd space before the `biosample_summary` in that case. Like before, you can just put a space before the previous `</span>` and it should work in all cases.

<img width="582" alt="biosample summary" src="https://user-images.githubusercontent.com/1181324/60533229-ed8c8880-9cb3-11e9-948c-d0fad47eb4f9.png">

Reply:
Thanks